frontend_tests/node_tests/navbar_alerts.js

```diff
@@ -35,7 +35,7 @@ set_global("document", {
 });
 
 const {localstorage} = zrequire("localstorage");
-const panels = zrequire("panels");
+const panels = zrequire("navbar_alerts");
```

Comment:
Should I also add a commit to rename `panels` with `navbar_alerts`?

Reply:
We should rename all the related JS variables and CSS classes. Squashing this all into one commit would be fine.